app/models/forum/post.js

```diff
@@ -1,19 +1,21 @@
-import Model, { belongsTo, attr } from '@ember-data/model';
 import { computed } from '@ember/object';
+import Model, { belongsTo, attr } from '@ember-data/model';
 
-export default Model.extend({
+export default class Post extends Model {
   // Properties
-  message: attr('string'),
-  messageCamofied: attr('string'),
-  createdAt: attr('date'),
-  updatedAt: attr('date'),
+  // @attr message;
+  @attr messageCamofied;
+  @attr createdAt;
+  @attr updatedAt;
 
   // Relations
-  author: belongsTo('user'),
-  thread: belongsTo('forum/thread'),
+  @belongsTo('user') author;
+
+  @belongsTo('forum/thread') thread;
 
   // Computed properties
-  isUpdated: computed('createdAt', 'updatedAt', function() {
+  @computed('createdAt', 'updatedAt')
```

Comment:
(Why) do you need to explicitly give these variables here? Isn't it the same as, for example, `userIsCurrentlyMember` in membership.js?